<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAttend</class>
 <widget class="QMainWindow" name="FaceAttend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceAttend</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="videowidget" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>481</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="widgetLb" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgba(49, 50, 54, 69);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>21</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="image.qrc">:/new/prefix1/img/yes.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 25 24pt &quot;微软雅黑 Light&quot;;
border:none;
background-color: rgba(49, 50, 54,0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>认证成功</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="headpicture">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>266</width>
       <height>266</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/new/prefix1/img/crc.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="videoLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>videoLb</zorder>
    <zorder>widgetLb</zorder>
    <zorder>headpicture</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49, 0.9);
}
QWidget#label_title{
	
	font: 24pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}
QWidget#label_headLb
{
	border-radius:75px;
}</string>
    </property>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>人脸识别考勤系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_headLb">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>211</y>
       <width>276</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49, 0.9);
}
QLabel
{
	border:none;	
	color: rgb(125, 100, 127);
	
	font: 16pt &quot;微软雅黑&quot;;
}
QLineEdit
{   
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:none;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>工号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numberEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>270</y>
       <width>276</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49, 0.9);
}
QLabel
{
	border:none;	
	color: rgb(125, 100, 127);
	
	font: 16pt &quot;微软雅黑&quot;;
}
QLineEdit
{   
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:none;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>姓名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>329</y>
       <width>276</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49, 0.9);
}
QLabel
{
	border:none;	
	color: rgb(125, 100, 127);
	
	font: 16pt &quot;微软雅黑&quot;;
}
QLineEdit
{   
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:none;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>部门：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="departmentEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>387</y>
       <width>276</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	border:1px solid #313236;
	border-radius:4px;
	
	background-color: rgba(37, 40, 49, 0.9);
}
QLabel
{
	border:none;	
	color: rgb(125, 100, 127);
	
	font: 16pt &quot;微软雅黑&quot;;
}
QLineEdit
{   
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:none;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>label_title</zorder>
    <zorder>label_headLb</zorder>
    <zorder>widget_5</zorder>
    <zorder>widget_6</zorder>
    <zorder>widget_7</zorder>
    <zorder>widget_4</zorder>
   </widget>
   <zorder>widget_3</zorder>
   <zorder>videowidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
